### PR TITLE
Create base storybook

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
image

### CODE REVIEW

File: src/components/Breadcrumb/Breadcrumd.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Breadcrumd from './Breadcrumd';
+
+export default {
+  component: Breadcrumd,
+} as Meta;
+
+type Story = StoryObj<typeof Breadcrumd>;
+
+export const primaryBreadcrumd: Story = {
```
Comment (hungdo-agilityio) [inline src/components/Breadcrumb/Breadcrumd.stories.tsx]: Do we have secondary Breadcrumb? Please recheck for all, if there was 1 variant use Default is good enough

File: pnpm-lock.yaml
Comment (hungdo-agilityio) [inline pnpm-lock.yaml:1]: Why this file change

File: src/components/Breadcrumb/Breadcrumd.stories.tsx (hunk 1 above)
Comment (hungdo-agilityio) [reply src/components/Breadcrumb/Breadcrumd.stories.tsx]: Recheck convention Story should be PascalCase

Comment (thanhtrantruong47) [reply src/components/Breadcrumb/Breadcrumd.stories.tsx]: Fixed

File: pnpm-lock.yaml
Comment (thanhtrantruong47) [reply pnpm-lock.yaml:1]: Fixed

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**